project-p3-forms-master/code/style.css

```diff
@@ -0,0 +1,332 @@
+.hamburger-dropdown-menu .toggler:checked + .hamburger > div:after {
+  top: 0;
+  transform: rotate(90deg);
+}
+
+/*** Show dropdown Menu ***/
+
+.hamburger-dropdown-menu .toggler:checked ~ .menu {
+  visibility: visible;
+}
+
+.hamburger-dropdown-menu .toggler:checked ~ .menu > div {
+  transform: scale(1);
+  transition-duration: 0.75s;
+}
+
+.hamburger-dropdown-menu .toggler:checked ~ .menu > div > div {
+  opacity: 1;
+  transition:  opacity 0.4s ease 0.4s;
+}
+
+.hamburger-dropdown-menu .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100;
+  visibility: hidden;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.hamburger-dropdown-menu .menu > div {
+  background: #24292d;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: flex-start;
+  justify-content: left;
+  padding-left: 20px;
+  padding-top: 40px;
+  transform: scale(0);
+  transition: all 0.4s ease;
+}
+
+.hamburger-dropdown-menu .menu > div > div {
+  text-align: left;
+  max-width: 90vw;
+  max-height: 100vh;
+  opacity: 0;
+  transition: opacity 0.4s ease;
+}
+
+.hamburger-dropdown-menu .menu > div > div > ul > li {
+  list-style: none;
+  color: #FFF;
+  font-size: 30px;
+  padding: 5px;
+}
+
+.hamburger-dropdown-menu .menu > div > div > ul > li > a {
+  color: inherit;
+  text-decoration: none;
+  transition: color 0.4s ease;
+}
+
+/*** Tablet/Desktop Menu ***/
+
+.desktop-menu {
+display: none;
+}
+
+.desktop-menu a {
+  color: #fff;
+  text-decoration: none;
+  padding: 17px 0;
+  margin-right: 16px;
+}
+
+.desktop-menu a:hover {
+  text-decoration: underline;
+}
+
+/*** About Container ***/
+
+.about-container {
+  background: #046F9E;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Essays Container ***/
+
+.essays-container {
+  background: #D3D6D7;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Letters Container ***/
+
+.letters-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Newsletter Container ***/
+
+.newsletter-container {
+  background: #D3D6D7;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Newsletter Sign Up Form ***/
+
+fieldset {
+  border: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+fieldset input {
+  border: transparent;
+  line-height: 25px;
+  margin: 5px 0;
+}
+
+fieldset input[type=text], [type=email] {
+  width: 100%;
+}
+
+fieldset span {
+  margin-left: 5px;
+}
+
+fieldset button {
+  margin-top: 5px;
+  width: 20%;
+}
+
+/*** Form Validation ***/
+
+fieldset input[type=text], [type=email]:valid {
+  background-color: #cfe3ff;
+}
+
+fieldset input[type=text], [type=email]:invalid {
+  border-color: crimson;
+}
```

Comment:
For these to work I think you need to write out input every time, like this:
```
fieldset input[type=text]:valid, input[type=email]:valid {
  background-color: #cfe3ff;
}

fieldset input[type=text]:invalid, input[type=email]:invalid {
  border-color: crimson;
}
```

(Same goes with valid and invalid, although I'm not sure your text input will ever be invalid. Try adding [min or max length](https://www.w3schools.com/tags/att_input_minlength.asp) and see if that works?)